cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -412,13 +412,16 @@ instance Show peer => Transformable Text IO [TraceLabelPeer peer
   trTransformer _ verb tr = trStructured verb tr
 
 -- transform @BlockFetchDecision@
-instance Show peer => Transformable Text IO (TraceLabelPeer peer
-                                (TraceFetchClientState header)) where
+instance {-# OVERLAPS #-}
```

Comment:
why does this type overlap with another? and with which? can they be put together in the file?
what caused you to introduce overlapping types?

Reply:
That's a bit odd.. I just dropped the overlaps and all is well now.  I clearly remember needing them.

Gone, anyway.